pkgs/games/osu-lazer/bin.nix

```diff
@@ -3,26 +3,27 @@
 , fetchurl
 , fetchzip
 , appimageTools
+, makeWrapper
 }:
 
 let
   pname = "osu-lazer-bin";
-  version = "2024.817.0";
+  version = "2024.906.1";
```

Comment:
Can you upgrade to 2024.906.2?
https://github.com/ppy/osu/releases/tag/2024.906.2